### PR TITLE
feat(Pipedrive Node): Add sort field for get all persons

### PR DESCRIPTION
**Pull request by daniel-alba17**
## Summary
Include sort field in Pipedrive get all persons node

image

### CODE REVIEW

**Review by Joffcom — APPROVED** [review]

**Comment by daniel-alba17** [comment]
@Joffcom Done!

**Review by Joffcom — APPROVED** [review]

**Review by netroy — CHANGES_REQUESTED** [review]
Sorry we forgot to merge this after the last approval.
Can you please merge the latest master back into your branch 🙏🏽 

---
Also, for any future PRs, **please keep this checkbox enabled**, so that we can rebase the PR without having to bother you.
 
image

**Review by netroy — APPROVED** [review]

**Pull request merged**
